docs/Platform_Development.rst

```diff
@@ -1786,35 +1848,35 @@ or the default ones in the system, it is stored under the **calendars**
-| exo:calendarColor     | String          | false      | The color name of the calendar that is defined in the **org.exoplatform.web ui.form.ext.UIFormColorPicker class (such as Sky blue, Powder blue).**   |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-| exo:calendarOwner     | String          | false      | The name of the user creating the calendar.                                                                                                          |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-| exo:locale            | String          | false      | Location where the calendar is set in format of the uppercase ISO 3166 3-letter country code.                                                        |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-| exo:timeZone          | String          | false      | The Id of the time zone that is set by the user in compliance with the Java class: java.util.TimeZone.                                               |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-| exo:publicUrl         | String          | false      | The public ICAL link of the calendar.                                                                                                                |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-| exo:privateUrl        | String          | false      | The private ICAL link of the calendar.                                                                                                               |
-+-----------------------+-----------------+------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
++-----------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field                 |          type   |  Description                                                                                                                                         |
```

Comment:
The calendar application is still in JCR, so we should not update this table IMO.

Reply:
I did not change it, it looks like a bad copy paste of the table heading

Reply:
ok so you will revert it ?
It looks like a column has been removed also (Multiple).

Reply:
The table structure was affected so it appears that the whole table was removed then added again (you notice this when you open whole file)

Reply:
It should be OK now